content/applications/hr/recruitment.rst

```diff
@@ -1,10 +1,827 @@
+- :guilabel:`Default Vehicle`: Using the drop-down menu, select the vehicle that will be assigned
+  to the applicant.
+- :guilabel:`Canteen Cost`: Enter the amount that is offered to the applicant for the cafeteria.
+- :guilabel:`Contract Type`: Using the drop-down menu, select the type of contract that is being
+  offered.
+
+.. image:: recruitment/generate-link.png
+   :align: center
+   :alt: Format the offer to send to the applicant. Make any modifications before sending.
+
+Send offer
+~~~~~~~~~~
+
+To send the offer to the applicant, click :guilabel:`Send`. If the applicant does not have an email
+address listed on their applicant card, the applicant form appears so their email can be configured.
+If there is already an email configured on the applicant's card, the email pop-up appears instead.
+If prompted with the applicant's card, enter the applicant's email address in the :guilabel:`Email`
+field, then click :guilabel:`Save`. The applicant form closes and the email pop-up appears.
+
+The :guilabel:`Applicant: Your Salary Package` email template is used, and the
+:guilabel:`Recipients`, :guilabel:`Subject`, and email body are pre-populated based on the email
+template. If any attachments need to be added, click the :guilabel:`Attachments` button and a file
+explorer window appears. Navigate to the desired file, then click :guilabel:`Open` to attach it to
+the email. The attachment loads, and is listed above the :guilabel:`Attachments` button. Once the
+email is ready to send, click :guilabel:`Send`.
+
+.. note::
+   To send an offer, ensure the *Sign* application is installed. This is necessary so the offer can
+   be sent to the applicant by the recruiter. The applicant does not need any software installed.
+
+.. image:: recruitment/send-offer.png
+   :align: center
+   :alt: Send an email to the applicant with a link to the offered salary.
+
+.. _signed:
+
+Contract signed
+---------------
+
+Once the applicant has accepted the offer and signed the contract, the next step is to move the
+applicant to the :guilabel:`Contract Signed` stage. To move the applicant to the next stage, drag
+and drop the applicant's card to the :guilabel:`Contract Signed` stage, or click the
+:guilabel:`More` button at the top of the applicant's card to reveal the :guilabel:`Contract
+Signed` stage button at the top of the individual applicant's card, and click :guilabel:`Contract
+Signed`.
+
+The :guilabel:`Contract Signed` stage is folded in the kanban view, but the card may still be
+dragged and dropped into that stage.
+
+Once the applicant's card moves to the :guilabel:`Contract Signed` stage, a green :guilabel:`Hired`
+banner appears in the top right of the applicant's card.
+
+.. image:: recruitment/hired.png
+   :align: center
+   :alt: A large green "HIRED" banner appears in the top right corner of an applicant's card after
+         they are moved ot the Contract Signed stage.
```

Comment:
Typo and no need for the alt tag to restate the previous paragraph- it can just describe the image :)
```suggestion
   :alt: Hired banner in the top right corner of applicant card.
```